.github/workflows/beaker.yml

```
@@ -162,6 +162,10 @@ jobs:
     name: "${{ matrix.name }}"
     steps:
       - uses: actions/checkout@v4
+      - name: disable apparmor
+        run: |
+          sudo aa-teardown || true
```

I like comments in code pointing to exact bugs
```suggestion
          # https://bugs.launchpad.net/ubuntu/+source/apparmor/+bug/2093797
          sudo aa-teardown || true
```

---

Added, thanks!